server/services/v1/auth_provider.go

```diff
@@ -341,3 +455,31 @@ func readDate(dateStr string) int64 {
 	}
 	return result
 }
+
+func createAccessTokenMetadataKey(clientSecret string) string {
+	hash := sha256.Sum256([]byte(clientSecret))
+	encodedHash := base64.StdEncoding.EncodeToString(hash[:])
+	return access_token_using_client_creds_prefix + encodedHash
+}
+
+func deleteApplicationMetadata(ctx context.Context, appId string, a *Auth0) error {
+	// delete metadata related to this app from user metadata
+	tx, err := a.txMgr.StartTx(ctx)
+	if err != nil {
+		log.Warn().Err(err).Msgf("Failed to start tx to cleanup metadata")
```

Comment:
Can be just `Msg(` not `Msgf(`
same below.

Reply:
changed msgf --> msg.